source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -0,0 +1,465 @@
+=================
+Server Monitoring
+=================
+
+:Spec: 1580
+:Title: Server Monitoring
+:Author: Shane Harvey
+:Advisors: Matt broadstone
+:Status: Accepted
+:Type: Standards
+:Version: 2.15
+:Last Modified: 2020-02-20
+
+.. contents::
+
+--------
+
+Abstract
+--------
+
+This spec defines how a driver monitors a MongoDB server. In summary, the
+client monitors each server in the topology. The scope of server monitoring is
+to provide the topology with updated ServerDescriptions based on isMaster
+command responses.
+
+META
+----
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in `RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_.
+
+Specification
+-------------
+
+Terms
+'''''
+
+See the terms in the `main SDAM spec`_.
```

Comment:
Yep. And I just fixed the links so that they work as intended.

This is a relative link:
```
.. _main SDAM spec: source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst
```

This is an absolute  link:
```
.. _main SDAM spec: /source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst
```